.github/workflows/build-publish.yml

```diff
@@ -104,7 +104,7 @@ jobs:
     permissions:
       contents: read
       id-token: write
-    uses: smartcontractkit/.github/.github/workflows/reusable-docker-build-publish.yml@16a9ca3a2352bcb47539ebbb6f6153d657f9f8d0 # 2025-03-31
+    uses: smartcontractkit/.github/.github/workflows/reusable-docker-build-publish.yml@dx-328/manual-dispatch-reusable # TODO
```

Comment:
Update all refs to SHA when this [PR](https://github.com/smartcontractkit/.github/pull/998) is merged.